vertical-pod-autoscaler/e2e/vendor/k8s.io/autoscaler/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1/types.go

```diff
@@ -147,6 +147,16 @@ type ContainerResourcePolicy struct {
 	// for the container. The default is no maximum.
 	// +optional
 	MaxAllowed v1.ResourceList `json:"maxAllowed,omitempty" protobuf:"bytes,4,rep,name=maxAllowed,casttype=ResourceList,castkey=ResourceName"`
+
+	// Specifies the type of recommendations that will be computed
+	// (and possibly applied) by VPA.
+	// If not specified, the default of [ResourceCPU, ResourceMemory] will be used.
+	ControlledResources *[]v1.ResourceName `json:"controlledResources,omitempty" patchStrategy:"merge" protobuf:"bytes,5,rep,name=controlledResources"`
+
+	// Whether autoscaler limit scaling is enabled for the container. The default is "Auto".
+	// Enabling this requires the autoscaler to be enabled for the container
+	// +optional
+	LimitMode *ContainerLimitScalingMode `json:"limitMode,omitempty" protobuf:"bytes,6,rep,name=limitMode"`
```

Comment:
I've thought about this a bit more and I think it would be good to make the setting here more explicit and change this to sth like:
ControlledValues: RequestsOnly/RequestsAndLimits

with RequestsAndLimits being the default.

WDYT?

Reply:
That reads better, updated :-)